paddle/fluid/operators/setitem_op.cc

```diff
@@ -0,0 +1,104 @@
+//   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "paddle/fluid/operators/setitem_op.h"
+
+#include <string>
+
+namespace paddle {
+namespace operators {
+
+class SetitemOp : public framework::OperatorWithKernel {
```

Comment:
Should we spell the class name as `SetItem`? That's much more like the C++ style. Also check the naming SetItem at other places and .h/.cc file names.

Reply:
Thanks, done